addOns/graphql/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 ### Changed
+- Update graphql-java dependency, with important bugfixes.
```

Comment:
There's already an entry mentioning the update, should be changed that instead, though not sure the fix is that relevant for how the add-on is used.

Reply:
An important fix was made in version 19.
According to the release notes:

https://github.com/graphql-java/graphql-java/releases/tag/v19.0

So when we import the graphql schema via URL, we get an invalid introspection query error due to the absence of the "isDeprecated" field.

Reply:
All fixes are important (especially if they affect oneself), stating that does not adding anything relevant to the release notes (stating the issues fixed would but usually we don't add those details).